pkg/parse/run.go

```diff
@@ -63,6 +64,8 @@ const (
 	RenderingNotRequired string = "Rendering not required but is currently enabled"
 )
 
+var retryBackoff = util.BackoffWithDurationLimit(10 * time.Minute)
```

Comment:
10 minutes seems like a long time to retry. I feel like the wait time we choose here should be a reasonable amount of time for retriable errors to resolve themselves, but if we wait too long then we delay:
- the error being surfaced on the RSync status
- the `Run` loop re-entering to trigger on the next timer

Granted it's a bit arbitrary, but I feel something like 1 minute might be closer to the right value. WDYT?

Reply:
Agreed. My original thought is to allow 10 internal retries before claiming it is not transient, but maybe too long. 

The retryBackoff is used twice, one for `SourceCommitAndDir` and the other one is for `readHydratedDir`.
I'm thinking we can use 5 minutes for `SourceCommitAndDir`, which allows 2 git-sync loops to be executed (based on a 120s timeout for one sync), and use 1 minute for `readHydratedDir`.

Reply:
changed to 5 mins and 1 min